src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -39,6 +48,41 @@ class HttpGoogleCloudStorageDAO(clientSecretsJson: String, dataStoreFactory: Dat
      .build()
   }
 
+  override def createGoogleGroup(userId: String, accessLevel: String, workspaceName: WorkspaceName, bucketName: String) = {
+    val credential = new GoogleCredential.Builder()
+      .setTransport(httpTransport)
+      .setJsonFactory(jsonFactory)
+      .setServiceAccountId(clientSecrets.getDetails.get("client_email").toString)
+      .setServiceAccountScopes(directoryScopes)
+      .setServiceAccountUser(clientSecrets.getDetails.get("sub_email").toString)
+      .setServiceAccountPrivateKeyFromP12File(new java.io.File("/test.broad.p12")) //add this file to jenkins?
+      .build()
+
+    val directory = new Directory.Builder(httpTransport, jsonFactory, credential).setApplicationName("firecloud:rawls").build()
+    val group = new Group().setEmail(s"FC-${workspaceName.namespace}_${workspaceName.name}-${accessLevel}@test.broadinstitute.com").setName(s"FireCloud ${workspaceName.namespace}/${workspaceName.name} ${accessLevel}")
+    val member = new Member().setEmail(userId).setRole(GCSAccessLevel.toGoogleString(GCSAccessLevel.Owner))
+
+    directory.groups().insert(group).execute()
+    directory.members().insert(s"FC-${workspaceName.namespace}_${workspaceName.name}-${accessLevel}@test.broadinstitute.com", member).execute()
```

Comment:
test.broadinstitute.com is the google apps domain that we have for now while we test this stuff. I should probably read that domain from the config file btw.

Also, it's actually .com which confused me for the longest time because I was trying to create groups under that domain